Dynamic tensor mechanics action enhancement

recuero opened the pull request:
## Reason
Make dynamic tensor mechanics action more user friendly and provide regression testing.

## Design
Follow existing design

## Impact
Better tested code, lower entry point to new or occasional users.

Closes #18388

recuero committed:
Make dynamic tensor mechanics action parameters more descriptive.

Add various tests checking for consistency in the dynamic tensor mechanics action.

Refs. #18388

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/779988/) on 8b1608d wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/18392/site)

This comment will be updated on new commits.

recuero committed:
Allow legacy parameter names in dynamic tensor mechanics action (#18388)

recuero committed:
Fix tests and allow pressure action to take hht_alpha. (#18388)

 -  Fix rayleigh_damping tests, which appear to want to check stress_yy (due to loading)
    but were outputting stress_xy. All the tests in that folder were regolded with
    the new results.
 -  Allow pressure action to take hht_alpha (in addition to deprecated alpha), in order
    to make it consistent with the dynamic tensor mechanics actions.

dschwen commented on modules/tensor_mechanics/test/tests/dynamics/rayleigh_damping/rayleigh_hht_ti.i:
```diff
@@ -39,186 +39,184 @@
-      alpha  = 0.11
-    [../]
-  [../]
+      alpha = 0.11
```
```suggestion
      hht_alpha = 0.11
```

recuero committed:
Update deprecated hht alpha parameter

Co-authored-by: Daniel Schwen <daniel@schwen.de>
